packages/block-editor/src/components/responsive-block-control/test/index.js

```diff
@@ -91,9 +91,9 @@ describe( 'Basic rendering', () => {
 
 		const toggleState = container.querySelector( 'input[type="checkbox"]' ).checked;
 
-		const defaultControlGroup = container.querySelector( '.block-editor-responsive-block-control__group--default' );
+		const defaultControlGroup = container.querySelector( '.block-editor-responsive-block-control__group:not(.is-responsive)' );
```

Comment:
It's a bummer that this requires the `:not` pseudo-selector. I suppose it's not particularly an issue here, but selectors that might potentially have several modifiers will need extra care; scoping a selector for a component's "default" state could require multiple `:not` checks that could get difficult to manage. Hopefully that's a relatively rare case though.

Reply:
(And I suppose it's more likely to be the case in tests like this; it's probably possible to architect without it being a concern in the CSS itself, and especially if some of the Emotion explorations move forward.)

Reply:
Yeah, I see this as mostly a use-case we'd want to isolate within tests, where we're explicitly wanting to distinguish between the default and the variant.

In real-world application, we should typically want to treat the variant as something of an additive _layer upon the default_, even to the extent of unsetting/overriding properties from those defaults, vs. ever feeling the need to use `:not( .is-variant )` in CSS. If we found ourselves needing to use `:not`, it probably raises a bigger question of whether we can really consider there to be a default (vs. other named variants, or treating them as wholly different elements).